### ISSUE TITLE
Session is expiring immediately

### ISSUE THREAD

**Issue by keloin** [dhleong/pepper-mint#7]
installed it as-is, wrote the code provided in the example, set my login/password, thats what I am getting:

```
$ node index.js 
Logged in...
Boo :( [Error: Session has expired]
```

**Comment by keloin** [comment]
authentication is broken, here is a very deep root cause analysis: https://github.com/mrooney/mintapi/issues/73

**Comment by ramonfritsch** [comment]
I'd suggest to use phantomjs for the log in method as they changed it very deeply.

**Comment by dhleong** [comment]
@jmbeach provided support for the new login mechanics requiring a `cookie` parameter. Just released as v1.0.0

If you still have issues with this, please comment and I'll reopen.
